e2e/pages/organizations/developers/developers.spec.js

```diff
@@ -41,6 +69,96 @@ describe('the Developers page', () => {
         it('should not have friendly name text box under POC', () => {
           expect(contact.friendlyName.isDisplayed()).toBe(false);
         });
+
+        it('should not allow to edit developer status', () => {
+          expect(page.developerStatus.isDisplayed()).toBe(false);
+        });
+
+        it('should allow editing of POC', () => {
+          contact.set(poc);
+          page.editWebsite.setValue(website);
+          actionBar.save();
+          expect(toast.toastTitle.getText()).toEqual('Update processing');
+          hooks.waitForSpinnerToDisappear();
+          expect(page.developerContact).toHaveTextContaining(poc.full);
+          expect(page.developerContact).toHaveTextContaining(poc.title);
+          expect(page.developerContact).toHaveTextContaining(poc.phone);
+          expect(page.developerContact).toHaveTextContaining(poc.email);
+          expect(page.developerWebsite).toHaveTextContaining(website);
+        });
+      });
+      describe('when looking at developer with more than one product', () => {
```

Comment:
We usually put blank lines between blocks
```suggestion
      });

      describe('when looking at developer with more than one product', () => {
```